src/Test/Utilities/Shared/Diagnostics/DiagnosticDescription.cs

```diff
@@ -391,7 +389,7 @@ public static string GetAssertText(DiagnosticDescription[] expected, IEnumerable
 
             // Write out the 'command line compiler output' including squiggles (easy to read debugging info in the case of a failure).
             // This will be useful for VB, because we can't do the inline comments.
-            if (includeCompilerOutput)
+#if false
```

Comment:
If that's the right thing then yes.  We shouldn't be checking in `#if false` code.